tests/integrationtest/r/expression/uuid.result

```diff
@@ -28,6 +28,48 @@ NULL
 SELECT UUID_TIMESTAMP('019b1440-87b7-7380-ab00-ce413e795004');
 UUID_TIMESTAMP('019b1440-87b7-7380-ab00-ce413e795004')
 1765571332.023000
+SELECT UUID_VERSION('00000000-0000-0000-0000-000000000000');
```

Comment:
Is this from the other PR?

Reply:
Yes, I'll set this PR to draft until the other PR is merged.